LOADERS/CLOUD_DATABASE/LOADER/LOADER.py

```diff
@@ -17,7 +17,7 @@
 
 @flojoy
 def LOADER(dc_inputs: list[DataContainer], params: dict):
-    api_key = get_api_key()
+    api_key = get_frontier_api_key()
     measurement_uuid = params["measurement_uuid"]
 
     if api_key != "" and measurement_uuid != "":
```

Comment:
The return value of `get_frontier_api_key()` would be either a string or None, so we need to change this checking to `if api_key is not None and ....`